### PR TITLE
feat: Responsive vault card skeleton loader

### LINKED ISSUE #928: Add responsive skeleton Loader component

### PR DESCRIPTION
**Pull request by talwinder50**
https://user-images.githubusercontent.com/7862595/129115923-ea578293-992f-4d17-b5cb-96d862201cea.mp4

Screen Shot 2021-08-11 at 7 20 08 PM
closes #928

Signed-off-by: talwinder kaur <talwinder.kaur@securekey.com>

#943 is created to address other two type credential and credential type

### CODE REVIEW

File: cmd/wallet-web/tailwind.js [hunk 1]
```diff
@@ -406,7 +408,82 @@ module.exports = {
...
+            paddingRight: theme('spacing.6'),
+            paddingBottom: theme('spacing.6'),
+            width: 256,
+            height: 160,
+            flexDirection: 'column',
+          },
+        },
+        '.skeleton-photo': {
+          backgroundColor: theme('colors.neutrals.selago'),
+          borderRadius: 9999,
```
Comment (birtony) [inline cmd/wallet-web/tailwind.js:435]: Let's use `'50%'` instead

File: cmd/wallet-web/tailwind.js [hunk 2]
```diff
@@ -406,7 +408,82 @@ module.exports = {
...
+          width: 48,
```
Comment (birtony) [inline cmd/wallet-web/tailwind.js:436]: We can use value from the theme spacing here

File: cmd/wallet-web/tailwind.js [hunk 3]
```diff
@@ -406,7 +408,82 @@ module.exports = {
...
+          '&::after': {
+            bottom: 0,
+          },
+
+          '& .top': {
+            width: '100%',
+          },
+
+          '& .bottom': {
+            width: '60%',
```
Comment (birtony) [inline cmd/wallet-web/tailwind.js:473]: This is intentional, right?

File: cmd/wallet-web/tailwind.js (hunk 2 above)
Comment (talwinder50) [reply cmd/wallet-web/tailwind.js]: updated

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**